docs/user/concepts/candidates-and-ambiguity.md

```
@@ -0,0 +1,147 @@
+from paxman.core.domain import Resolution
+
+result = paxman.canonicalize("01/02/2026", Date.create_contract())
+if result.status == Resolution.AMBIGUOUS:
+    for c in result.candidates:
+        p = c.provenance[0]
+        print(f"  {c.value!r} via {c.validation_rule} ({p.authority}: {p.specification_name}) span={c.span}")
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)